x-pack/plugins/security_solution/public/resolver/data_access_layer/mocks/one_ancestor_two_children.ts

```diff
@@ -43,24 +40,11 @@ interface Metadata {
 /**
  * A simple mock dataAccessLayer possible that returns a tree with 0 ancestors and 2 direct children. 1 related event is returned. The parameter to `entities` is ignored.
  */
-export function oneAncestorTwoChildren(
-  { withRelatedEvents }: { withRelatedEvents: Iterable<[string, string]> | null } = {
```

Comment:
Removed options from this function, split that behavior off into its own function.